### PR TITLE
feat: resolve the file ext automatically

### PR DESCRIPTION
**Pull request by Kinbaum**

### CODE REVIEW

**Review by guimkogus — COMMENTED** [review]
a

File: src/index.js [hunk 1]
```diff
@@ -95,25 +95,47 @@ module.exports = class FederatedTypesPlugin {
...
+
+  extractTypes() {
+    const normalizedFileNames = Object.values(this.exposedComponents)
+      .map(exposed => {
+        const [ rootDir, entry ] = exposed.split(/\/(?=[^/]+$)/);
+        const ext = this.getExtension(rootDir, entry);
+
+        return path.resolve(process.cwd(), rootDir, ext);
+      })
+      .filter(entry => /\.tsx?$/.test(entry));
```
Comment (guimkogus) [inline src/index.js:138]: If I understood it right, I guess we'd like to also include .ts files in here:
```suggestion
      .filter(entry => /\.(ts|tsx)?$/.test(entry));
```

Comment (Kinbaum) [reply src/index.js:138]: The regex I’ve added handles both ts and tsx extensions. No changes are necessary

**Comment by Kinbaum** [comment]
@pavandv when you have a chance, can you review this

**Comment by pavandv** [comment]
I'll look into this today.

**Review by pavandv — APPROVED** [review]

**Pull request merged**